src/lib/buttons/default.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  font-size: 1em;
+  font-family: Montserrat;
+  font-weight: 500;
+  text-align: center;
+  background-color: ${props => props.theme.props[0]};
```

Comment:
Let's make props an object (with a defined type?). props[0] - you'll kill yourself in a half of year

Reply:
:D agree, done.